### PR TITLE
Fixed SVG file image view

### PR DESCRIPTION
**Pull request by Azzamjoy44**
### Pull Request Overview

This pull request enables the project architecture .svg image file to be seen well in both light mode and dark mode

### Testing Strategy

This pull request was tested by Zafar Azzam

### TODO or Help Wanted

This pull request still needs verification that everything is good

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Zafar Azzam <azzam.zafar@stud.fils.upb.ro>

### CODE REVIEW

File: docs/project/2024/azzam.zafar/project_architecture_dark_mode.svg
Comment (alexandruradovici) [inline docs/project/2024/azzam.zafar/project_architecture_dark_mode.svg:1]: Please export the SVG file, this is a draw.io file.

File: docs/project/2024/azzam.zafar/project_architecture_light_mode.svg
Comment (alexandruradovici) [inline docs/project/2024/azzam.zafar/project_architecture_light_mode.svg:1]: Please export the SVG file, this is a draw.io file.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/azzam.zafar/project_architecture_light_mode.svg
Comment (Azzamjoy44) [reply docs/project/2024/azzam.zafar/project_architecture_light_mode.svg:1]: how is this file a draw.io file? This file name does not have the .drawio extention written anywhere in it

File: docs/project/2024/azzam.zafar/project_architecture_dark_mode.svg
Comment (Azzamjoy44) [reply docs/project/2024/azzam.zafar/project_architecture_dark_mode.svg:1]: How is this file a draw.io file? This file name does not have the .drawio extention written anywhere in it

Comment (alexandruradovici) [reply docs/project/2024/azzam.zafar/project_architecture_dark_mode.svg:1]: Use the export function of draw.io.

**Review by alexandruradovici — APPROVED** [review]

File: docs/project/2024/azzam.zafar/project_architecture_dark_mode.svg
Comment (alexandruradovici) [reply docs/project/2024/azzam.zafar/project_architecture_dark_mode.svg:1]: This is still a draw.io file:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<!-- Do not edit this file with editors other than draw.io -->
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Still not valid images, please use the export function

**Review by alexandruradovici — APPROVED** [review]

**Review by alexandruradovici — APPROVED** [review]

File: docs/project/2024/azzam.zafar/project_architecture_dark_mode.svg
Comment (Azzamjoy44) [reply docs/project/2024/azzam.zafar/project_architecture_dark_mode.svg:1]: How is it now?

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Pull request merged**